lib/node.js

```diff
@@ -1372,6 +1372,25 @@ declare class https$Server extends tls$Server {
   timeout: number;
 }
 
+type http$requestOptions = {
+  agent?: boolean | http$Agent<net$Socket>,
+  auth?: string,
+  createConnection?: (options: net$connectOptions, callback?: Function) => net$Socket,
+  defaultPort?: number,
+  family?: number,
+  headers?: {[key: string] : string},
```

Comment:
Unless I'm missing it the docs don't seem to specify that `headers` only maps strings to strings--seems like you could have at least numbers too. Maybe `{[key: string] : mixed}`?

Reply:
✅ Mixed it is.